.azure-pipelines.yml

```diff
@@ -17,7 +17,9 @@ variables:
   winfsp.version: '1.8.20304'
   pytest.base_args: |
     --log-level=DEBUG \
-    --durations=10 -v \
+    --"durations"=10 \
```

Comment:
Why we need to quote `durations` ?

Reply:
![image](https://user-images.githubusercontent.com/3187637/169100297-7d58737e-5deb-417d-a048-e498619a6533.png)

Seriously, that's an error, I've fixed it :+1: